lib/actions/users.ts

```
@@ -0,0 +1,20 @@
+export const createUser = async (input: NewUserParams) => {
+  try {
+    const { name, bio } = insertUserSchema.parse(input);
+
+    const userString = `NAME: ${name}. BIO: ${bio}`;
+    const embedding = await getEmbeddings(userString);
+
+    await db.insert(users).values({ name, bio, embedding });
+
+    return { success: true };
+  } catch (error) {
+    throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)